test/models/auditing/PenaltyAppealFileNotificationStorageFailureModelSpec.scala

```diff
@@ -49,7 +57,7 @@ class PenaltyAppealFileNotificationStorageFailureModelSpec extends SpecBase with
 
     "show the correct audit details" in {
       model.detail shouldBe Json.obj(
-        "notifications" -> Json.toJson(notifications)
+        "notifications" -> Json.toJson(notifications)(SDESNotification.auditSeqWrites)
```

Comment:
(I know you didn't write the test initially, but) I'm thinking we could make this a bit more concrete as it's just 'going with the flow' at this point, like whatever you set auditSeqWrites to, it will just accept? Instead we can manually write the json object that it should look like?